cmd/fluxsvc/main.go

```diff
@@ -56,6 +58,25 @@ func main() {
 		os.Exit(0)
 	}
 
+	// If the events-url flag is present, ignore instance specific Slack settings.
+	var defaultEventsConfig *instance.Config
+	if eventsURL != nil {
+		defaultEventsConfig = &instance.Config{
+			Settings: service.InstanceConfig{
+				Slack: service.NotifierConfig{
+					Username:        "Weave Cloud",
```

Comment:
This name shouldn't be hard-coded.
Maybe this whole section should be read from a file which could be configured via ConfigMap?

Reply:
Notifications service ignores this field anyway. Removed.